CIP-1694/README.md

```diff
@@ -0,0 +1,1163 @@
+| 5. Protocol Parameter Changes                      | Any change to one or more updatable protocol parameters, excluding changes to major protocol versions ("hard forks") |
+| 6. Treasury Withdrawals                            | Movements from the treasury, sub-categorized into small, medium or large withdrawals (based on the amount of Lovelace to be withdrawn). The thresholds for treasury withdrawals are discussed below. |
+| 7. Info                                            | An action that has no effect on-chain, other than an on-chain record. |
+
+**Any Ada holder** can submit a governance action to the chain.
+They must provide a deposit of `govDeposit` Lovelace, which will be returned when the action is finalized
+(whether it is **ratified**, has been **dropped**, or has **expired**).
+The deposit amount will be added to the _deposit pot_, similar to stake key deposits.
+
+Note that a motion of no-confidence is an extreme measure that enables Ada holders to revoke the power
+that has been granted to the current constitutional committee.
+Any outstanding governance actions, including ones that the committee has ratified or ones that would be enacted this epoch,
+will be dropped if the motion is enacted.
+
+#### Ratification
+
+Governance actions are **ratified** through on-chain voting actions.
+Different kinds of governance actions have different ratification requirements but always involve **two of the three** governance bodies.
+Depending on the type of governance action, an action will thus be ratified when a combination of the following occurs:
+
+* the constitutional committee approves of the action (`quorum`-many members vote 'Yes')
+* the DReps approve of the action (the stake controlled by the DReps who vote 'Yes' meets a certain threshold of the total registered voting stake)
+* the SPOs approve of the action (the stake controlled by the SPOs who vote 'Yes' meets a certain threshold over the total registered voting stake)
+
+> **Warning**
+> As explained below, different stake distributions apply to DReps and SPOs.
+
+##### Requirements
+
+The following table details the ratification requirements for each governance action scenario. The columns represent:
+
+* **Governance action type**<br/>
+  The type of governance action. Note that the protocol parameter updates are grouped into three categories.
+
+* **Constitutional committee (abbrev. CC)**<br/>
+  A value of ✓ indicates that `quorum`-many constitutional committee 'Yes' votes are required.<br/>
+  A value of - means that constitutional committee votes do not apply.
+
+* **DReps**<br/>
+  The DRep vote threshold that must be met as a percentage of *active voting stake*.<br/>
+  A value of - means that DReps votes do not apply.
+
+* **SPOs**<br/>
+  The SPO vote threshold which must be met as a percentage of the stake held by all stake pools.<br/>
```

Comment:
Instead of being a percentage of stake held why not use pledge? 

We have seen the gamification with ISPOs moving delegation around or even incentivizing delegation. With pledge it would be a way to normalize the weight a SPO or pool group has when it comes to voting power. Pledge was intended to be a means to have "skin in the game" and a more significant way to secure the network. For reasons I am unaware of that was thrown out the window when the ITN ended. 

Exchanges and other relatively big companies/corporations tend to have significantly low if not 0 ADA as their pledge.

Reply:
Yes, we are considering this, see https://github.com/JaredCorduan/CIPs/blob/voltaire-v1/CIP-1694/README.md#pledge-weighted-spo-voting

Pure pledge is but one point on a spectrum.

Reply:
Rick had a twitter space not too long ago and this was one of my suggestions. Glad to see that it made it on the cip. Im sure i was not the only one with this idea.

Reply:
I included it when @AndrewWestberg suggested it at the workshop

Reply:
Percentage of stake held is a direct consequence of pledge in IOG research' rewards sharing scheme paper (https://iohk.io/en/research/library/papers/reward-sharing-schemes-for-stake-pools/). If the current system doesn't reflect that it means it is badly implemented.
Using pledge instead of percentage of stake wouldn't need to be considered if the staking mechanism was correctly implemented.

Reply:
It's also possible that the “game theory” in that RSS paper is just too simplistic and you will always have ADA holders who delegate based on popularity, on sympathy for small stake pools, or on mission-driven aspects without regarding pledge. That would mean that there cannot be a “correct” implementation achieving pledge and stake being correlated.

Reply:
> I included it when @AndrewWestberg suggested it at the workshop

Well good to see people independent of each other converge on the same idea. ;)

Reply:
> It's also possible that the “game theory” in that RSS paper is just too simplistic and you will always have ADA holders who delegate based on popularity, on sympathy for small stake pools, or on mission-driven aspects without regarding pledge. That would mean that there cannot be a “correct” implementation achieving pledge and stake being correlated.

A strict "correct" implementation can't be achieved in the real world yes, but one with bounded deviations can. Currently that deviation is much larger because delegating based on popularity, on sympathy for small stake pools, or on mission-driven aspects without regarding pledge has very little to no cost.

Reply:
How do you define very little to no cost?